src/GraphQL.Tests/NameFieldResolverTests.cs

```diff
@@ -16,6 +16,7 @@ public class NameFieldResolverTests
         [InlineData("naMe", "Anyone")]
         [InlineData("FullInfo", "Anyone 20")]
         [InlineData("fullInfo", "Anyone 20")]
+        [InlineData(null, null)]
```

Comment:
This looks like a case that should never happen.  You should never actually have a FieldAst with a null name.

Reply:
Understood, my intent was only to cover edge case, and to prove that it wont blowup with null ref exception, if you wish we can remove this

Reply:
This test is OK.